README.md

```diff
@@ -195,6 +195,10 @@ A list of companies that have paid Developer Community Writer Programs.
 
 - [WPHUB](https://www.wphub.com/write-for-us/) - $200 per piece
   > Wordpress tutorials and articles.
+
+- [Enlear](https://www.enlear.com/) - $50 to $150 per article
+  > Get paid to write about broad area of technical topics. Including Vue, Angular, React, Node, JS, DynamoDB
+  
```

Comment:
I opened an account here but my Enlear account is saying free. Do I need to contribute first before it becomes a paid status?

Reply:
yes

Reply:
Okay thanks

Reply:
thanks for adding the range!